distribution/changelog.txt

```diff
@@ -28,6 +28,7 @@
 - Fix: [#10705] Apply multithreaded rendering to all viewports.
 - Fix: [#10739] Mountain tool overlay for even-numbered selections.
 - Fix: [#10752] Mute button state not correctly set at startup.
+- Improved: [#682] The staff patrol area is now drawn on the water, instead of on the surface under water.
```

Comment:
I think you can file this as a feature, tbh. That, or move it up a bit, so it doesn't get lost in the sea of bugfixes.

Reply:
It's not really a feature, though, in that it doesn't give you any extra possibilities or even information - it just makes it easier to see.

Reply:
Alright. Semantics aside, a lot of people will like this change, though. I really think improvements ought to be higher up in the changelog.

Reply:
If this one were moved now it would mean that either all other improvements in the changefile needed to be moved or just that the ordering would be different than it always has been on this one.

Reply:
I'm fairly sure the way @Gymnasiast filed it is consistent with how it was done before. However, I believe that's a historical mistake that we should rectify at some point. :wink:

Reply:
I actually originally said the same about him following the conventions but edited my comment because I scrolled through the whole file and noticed that ordering and naming (some just say Improve) have changed over time once or twice.